catalog/istio-stack/config/serviceentries.yaml

```diff
@@ -0,0 +1,47 @@
+################################################################################
+# Google Service Entries
+################################################################################
+---
+apiVersion: networking.istio.io/v1beta1
+kind: ServiceEntry
+metadata:
+  name: external-google-metadata
```

Comment:
this is too much coupled on google. I wouldn't call it config then, instead do it like: https://github.com/kharf/flux-k8s-stack/tree/main/catalog/kube-prometheus-stack